resources/views/layouts/header.blade.php

```diff
@@ -20,7 +20,7 @@
 	<link rel="canonical" href="https://admin.netsoc.co" />
 	<meta property="og:title" content="UCC Netsoc Admin" />
 	<meta property="og:url" content="https://admin.netsoc.co" />
-	<meta property="og:image" content="http://files.netsoc.co/f/e9e0bee5b4/?dl=1" />
+	<meta property="og:image" content="https://files.netsoc.co/f/e9e0bee5b4/?dl=1" />
```

Comment:
Does the server require https? If not "://files.netsoc.co/f/e9e0bee5b4/?dl=1" might be more appropriate so that we use the same protocol as the referrer